.github/ISSUE_TEMPLATE/documentation-improvement.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Documentation improvement
+about: Suggest an improvement or report a bug in the documentation
+---
+
+<!-- Thank you for your contribution. Before you submit the issue:
+1. Search open and closed issues for duplicates.
+2. Read the contributing guidelines.
+3. Assign the Documentation project.
+-->
+
+**Description**
+
+<!-- Provide a clear and concise description of the potential documentation improvement.-->
+
+**Area**
```

Comment:
Do we need this part? It will always be either Lifecycle Manager or Modularization (which translates to the same thing basically). I would recommend dropping it. 🙂